docs/news.rst

```diff
@@ -3,6 +3,132 @@
 Release notes
 =============
 
+.. _release-2.3.0:
+
+Scrapy 2.3.0 (2020-0?-??)
+-------------------------
+
+Highlights:
+
+* :ref:`Feed exports <topics-feed-exports>` support :ref:`Google Cloud
+  Storage <topics-feed-storage-gcs>`
+* New :setting:`FEED_EXPORT_BATCH_ITEM_COUNT` setting for batch deliveries
+* The base implementation of :ref:`item loaders <topics-loaders>` has been
+  moved into :doc:`itemloaders <itemloaders:index>`
```

Comment:
This is again, a style preference, but I wonder if it is better to have a more detailed description of this feature in the highlights,to  highlight what is this change for. The same text as below can be used ("he base implementation of item loaders has been moved into a separate library, itemloaders, allowing usage from outside Scrapy and a separate release schedule")